fairlearn/postprocessing/_threshold_optimizer.py

```diff
@@ -65,12 +185,13 @@ class ThresholdOptimizer(BaseEstimator, ClassifierMixin, MetaEstimatorMixin):
     """
 
     def __init__(self, *, estimator=None,
-                 constraints=DEMOGRAPHIC_PARITY, grid_size=1000, flip=True,
+                 constraints=DEMOGRAPHIC_PARITY, objective=ACCURACY_SCORE, grid_size=1000, flip=True,
```

Comment:
the issue with having these constants such as `ACCURACY_SCORE` is that when user sees the autocomplete in their IDE, it would look less confusing if they saw `"accuracy_score"` I think.

Reply:
Fair point! Does sklearn repeat all the strings all over the place, or is there a better pattern that we just haven't come across?

Reply:
I guess it's just a norm that different communities follow. And in a dynamically typed lang like python, having the strings makes sense IMO. Alternatively, you could go for an `Enum`, but I haven't seen that often in python, and it can complicate things. What makes sense to me, is ['Literal`](https://www.python.org/dev/peps/pep-0586/) which we probably will see more of once people start adopting Python 3.8 standards.

Reply:
Moving to explicit strings since metric names are now stable.